resources/build/build-help.inc.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+#
+# This script contains utilities for converting markdown help documentation to htm/html
+# Expected platforms: android, ios, mac
+# Note: linux app doesn't need to convert from markdown to html
+#
+
+# Clean the htm/html files and recreate the output folder
+function _build_help_clean() {
+  echo "Cleaning"
+  if [[ "$1" == "android" ]]; then
+    DESTHTM="$KEYMAN_ROOT/android/KMAPro/kMAPro/src/main/assets/info"
+  elif [[ "$1" == "ios" ]]; then
+    DESTHTM="$KEYMAN_ROOT/ios/keyman/Keyman/Keyman/resources/OfflineHelp.bundle/Contents/Resources"
+  elif [[ "$1" == "mac" ]]; then
+    DESTHTM="$KEYMAN_ROOT/mac/Keyman4MacIM/Keyman4MacIM/Help"
+  fi
```

Comment:
> 
```
# * `output_path`  path to emit files to, relative to `$KEYMAN_ROOT/$platform`
```

Are you sure you want the paths relative to `$platform` instead of absolute paths?

Reply:
ok, I implemented with output_path as relative path.